bin/logstash

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
this should still be /bin/sh

Reply:
@jordansissel `bash` is more portable here imo :( Your code only works with `sh` `4.x` => `bash` `3.x` works fine though => standard OSX and old Linux will break if we go with `sh`   => `bash` wins, especially since JRuby requires `bash` anyways in the way we currently used to invoke it? :)

Reply:
Hmm, fair. I didn't realize that bin/jruby invoked bash.